utils/mock-data-loader/bss/deals/deals-to-submit-to-tfm.js

```diff
@@ -0,0 +1,242 @@
+    'supplier-correspondence-address-line-2': '',
+    'supplier-correspondence-address-line-3': '',
+    'supplier-correspondence-address-town': '',
+    'supplier-correspondence-address-postcode': '',
+    'industry-sector': {
+      code: '1005',
+      name: 'Construction',
+    },
+    'industry-class': {
+      code: '42910',
+      name: 'Construction of water projects',
+    },
+    'sme-type': 'Medium',
+    'supply-contract-description': 'TEST',
+    legallyDistinct: 'false',
+    'indemnifier-companies-house-registration-number': '',
+    'indemnifier-name': '',
+    'indemnifier-address-country': {},
+    'indemnifier-address-line-1': '',
+    'indemnifier-address-line-2': '',
+    'indemnifier-address-line-3': '',
+    'indemnifier-address-town': '',
+    'indemnifier-address-postcode': '',
+    'indemnifier-correspondence-address-country': {},
+    'indemnifier-correspondence-address-line-1': '',
+    'indemnifier-correspondence-address-line-2': '',
+    'indemnifier-correspondence-address-line-3': '',
+    'indemnifier-correspondence-address-town': '',
+    'indemnifier-correspondence-address-postcode': '',
+    indemnifierCorrespondenceAddressDifferent: '',
+    'buyer-name': 'A1 Test',
+    'buyer-address-country': {
+      name: 'United Kingdom',
+      code: 'GBR',
+    },
+    'buyer-address-line-1': 'TEST',
+    'buyer-address-line-2': 'TEST',
+    'buyer-address-line-3': 'TEST',
+    'buyer-address-town': 'TEST',
+    'buyer-address-postcode': 'TEST',
+    destinationOfGoodsAndServices: {
+      name: 'United Arab Emirates',
+      code: 'ARE',
+    },
+    supplyContractValue: '5000000.00',
+    supplyContractCurrency: {
+      text: 'GBP - UK Sterling',
+      id: 'GBP',
+      currencyId: 12,
+    },
+    supplyContractConversionRateToGBP: '',
+    'supplyContractConversionDate-day': '',
+    'supplyContractConversionDate-month': '',
+    'supplyContractConversionDate-year': '',
+  },
+  summary: {},
+  comments: [],
+  editedBy: [
+    {
+      date: '1626968578071',
+      username: 'maker1@ukexportfinance.gov.uk',
+      roles: [MAKER],
+      bank: {
+        id: '9',
+        name: 'UKEF test bank (Delegated)',
+        emails: ['checker@ukexportfinance.gov.uk'],
+      },
+      userId: '60f7d72654f99900074c0a6d',
+    },
+    {
+      date: '1626971072253',
+      username: 'maker1@ukexportfinance.gov.uk',
+      roles: [MAKER],
+      bank: {
+        id: '9',
+        name: 'UKEF test bank (Delegated)',
+        emails: ['checker@ukexportfinance.gov.uk'],
+      },
+      userId: '60f7d72654f99900074c0a6d',
+    },
+  ],
+  mandatoryCriteria,
+  supportingInformation: {
+    validationErrors: {
+      count: 0,
+      errorList: {
+        exporterQuestionnaire: {},
+      },
+    },
+    securityDetails: {
+      exporter: 'Test',
+    },
+  },
+  exporter: {
+    companyName: 'Auto Test 1',
+  },
+};
+
+const deals = new Array(25).fill({}).map((_item, index) => ({
+  ...dealDefaults,
```

Comment:
This could really do with either deep copy here or using immer to create this. (If you use immer, you'll want to make sure that it is a deep copy!)